packages/server/src/middleware/nuxt.js

```diff
@@ -122,7 +122,7 @@ const defaultPushAssets = (preloadFiles, shouldPush, publicPath, options) => {
   return links
 }
 
-const getCspString = ({ cspScriptSrcHashes, allowedSources, policies, isDev }) => {
+const getCspString = ({ cspScriptSrcHashes = [], allowedSources, policies, isDev }) => {
```

Comment:
This function should not be called in this case at all. I'll add some change

Reply:
Wouldn't there be a place where hashes were not defined, but other CSP policies is defined?
especially when https://github.com/nuxt/nuxt.js/pull/5387 is in, a simple `unsafe-inline` would cause `cspScriptSrcHashes` to be empty

Reply:
@williamchong007 we are checking `cspScriptSrcHashes` to be defined before calling `getCspString`. The default value of `cspScriptSrcHashes` is an empty array from `renderer.js`. (So still leads to calling this function)